src/theme/BlogPostItem/Footer/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { useColorMode } from '@docusaurus/theme-common';
+// @ts-expect-error exports do not work for some reason
+import { useBlogPost } from '@docusaurus/theme-common/internal';
+import type { WrapperProps } from '@docusaurus/types';
+import Giscus from '@giscus/react';
+import Footer from '@theme-original/BlogPostItem/Footer';
+import type FooterType from '@theme/BlogPostItem/Footer';
+import React from 'react';
+
+type Props = WrapperProps<typeof FooterType>;
+
+export default function FooterWrapper(props: Props): JSX.Element {
+  // Current page is a blog post page, or the blog list overview page
+  const { isBlogPostPage } = useBlogPost();
+
+  if (!isBlogPostPage) {
+    return <Footer {...props} />;
+  }
+
+  const { colorMode } = useColorMode();
+
+  return (
+    <>
+      <Footer {...props} />
+
+      <Giscus
+        repo="stryker-mutator/stryker-mutator.github.io"
+        repoId="MDEwOlJlcG9zaXRvcnk1OTE3NzI0Nw=="
+        category="Announcements"
```

Comment:
We should probably create a dedicated discussion category for this

Reply:
Are we going to have a lot of other announcements for the website? Or I guess it might be cleaner as a category